tests/ui/test_ui_tools.py

```diff
@@ -2211,6 +2211,52 @@ def test_main_view_generates_EDITED_label(
             assert label[0].text == "EDITED"
             assert label[1][1] == 7
 
+    @pytest.mark.parametrize(
+        "message",
```

Comment:
This is specifically a stream message, though the behavior shouldn't vary across message types. Can we add a note here regarding that we should refactor this? We have other tests with a single parametrize entry, but it's a little misleading.

Reply:
Yeah, I used basically it as base template, which I'm modifying as per the `to_vary_in_each_message` parameters. Surely I'll add a note here :+1:.